mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,45 @@
+import {ReactElement} from "react";
+
+/**
+ * Params for {@link Text}
+ */
+interface TextProps {
+
+  /**
+   * Displayed values
+   */
+  value: string | number;
+
+  /**
+   * Displayed placeholder
+   */
+  placeholder: string;
+
+  /**
+   * Function to render value
+   */
+  renderValue: (value: string | number) => ReactElement;
```

Comment:
You don't need to pass function as props in this case.

Reply:
Deleted the function